### ISSUE TITLE
Bridges

### ISSUE THREAD

**Issue by CrackersTie36** [Tribler/tribler#6063]
Like in the Tor browser the Tribler client should have something like "Bridges" for people that has a censored network

**Comment by devos50** [comment]
Thanks for your suggestion. If I understand this correctly, you suggest using Tribler as a generic proxy for data beyond torrent downloading?

**Comment by CrackersTie36** [comment]
No,  Do you know how Tor Browser have this feature called "Bridges" which allows you to connect to Tor on a network that blocks Tor, my idea is have something like that but on Tribler

**Comment by devos50** [comment]
Ah I see. Looks like a valuable addition to Tribler, but we currently have more important features to focus on. We will, however, keep your suggestion in the backlog and hopefully implement this functionality when we can 👍

**Comment by CrackersTie36** [comment]
Ok, Understandable plus respect for contributing to Tribler

**Comment by qstokkink** [comment]
With the new web-GUI of Tribler 8, it is possible to use HTTPS connections to access Tribler remotely (e.g., #8293). This means that you can rent a VPS (or set up a server) to perform Tribler traffic, while only interacting through HTTPS from your local machine. This is equivalent to a bridge.
